extension.ts

```diff
@@ -11,38 +11,83 @@ import { showCmdLine } from './src/cmd_line/main';
 import { ModeHandler } from './src/mode/modeHandler';
 import { TaskQueue } from './src/taskQueue';
 import { Position } from './src/motion/position';
+<<<<<<< aa8f6051a847c442e374ce22920ca5bd21835aec
+=======
+
+export class EditorIdentity {
+    private _fileName: string;
```

Comment:
The catch is we need to store both file name and column number and compare file name or both of them sometimes. If we merge file name and column number into one single string, we need to split it when we only need to compare file name. Here I found a separate class is more intuitive.